Dockerfile.oraclelinux7

```diff
@@ -161,3 +164,38 @@ RUN --mount=type=bind,source=.,target=rondb-src \
 # run with --output <output-folder>
 FROM scratch AS get-package-all
 COPY --from=build-all /root/rondb-tarball .
+
+##################################
+# Intermediate Image for RDRS ####
+##################################
+
+FROM ubuntu:22.04 AS rdrs-intermediate
+
+WORKDIR /root
+COPY --from=build-all /root/rondb-tarball .
+
+RUN mkdir -p /srv/hops/mysql/bin /srv/hops/mysql/lib /srv/hops/mysql/lib/private /srv/hops/mysql-cluster /srv/hops/mysql-cluster/log
+
+RUN --mount=type=bind,source=.,target=rondb-src \
+   mkdir /root/distro && \
+   tar xf *.tar.gz -C distro --strip-components=1 && \
+   cp distro/bin/rdrs /srv/hops/mysql/bin && \
+   cp distro/lib/libndbclient.so.6.1.0 /srv/hops/mysql/lib/ && \
+   cp distro/lib/librdrclient.so /srv/hops/mysql/lib/ && \
+   cp distro/lib/private/libcrypto.so.1.1 /srv/hops/mysql/lib/private && \
+   cp distro/lib/private/libssl.so.1.1 /srv/hops/mysql/lib/private && \
```

Comment:
Just use an env variable `MySQL_HOME_DIR` instead of retyping the PATH 8 types. Or make it the `WORKDIR`.